README.md

```diff
@@ -1,78 +1,96 @@
+</p>
 
-For a complete introductory example, see our [basic usage notebook](https://github.com/probabl-ai/skore/blob/main/examples/basic_usage.ipynb). The resulting skore report has been exported to [this HTML file](https://gist.github.com/augustebaum/6b21dbd7f7d5a584fbf2c1956692574e): download it and open it in your browser to visualize it.
+## ⚙️ Installation
+
+You can install `skore` by using `pip`:
+```bash
+pip install -U skore
+```
+
+## 🚀 Quick start
+=======
 
 In your shell, run the following to create a project file `project.skore` (the default) in your current working directory:
-```sh
-python -m skore create
+```bash
+python -m skore create 'project.skore'
 ```
 
-Run the following in your Python code to load the project:
+Run the following in your Python code (in the same working directory) to load the project, store some objects, delete them, etc:
 ```python
 from skore import load
 
+# load the project
 project = load("project.skore")
-```
 
-You can save items you need to track in your project:
-```python
+# save an item you need to track in your project
 project.put("my int", 3)
-```
 
-You can also get them back:
-```python
+# get an item's value
 project.get("my int")
-```
 
-By default, strings are assumed to be Markdown:
-```python
+# by default, strings are assumed to be Markdown:
 project.put("my string", "Hello world!")
-```
 
-Note that `put` overwrites previous data
-```python
+# `put` overwrites previous data
 project.put("my string", "Hello again!")
-```
 
-You can list all the keys in a project with:
-```python
-project.list_keys()
-```
+# list all the keys in a project
+print(project.list_item_keys())
 
-You can delete items with:
-```python
+# delete an item
 project.delete_item("my int")
 ```
 
-Then, in the directory containing your project, run the following command to start the UI locally:
-```sh
+Then, in the directory containing your project, run the following command in your shell to start the UI locally:
+```bash
 python -m skore launch project.skore
 ```
-
 This will automatically open a browser at the UI's location.
+In the `Elements` tab on the left, you can visualize the stored items.
+Create a new `View`, then you can then add items into this view.
+
+💡 Note that after launching the dashboard, you can keep modifying current items or store new ones, and the dashboard will automatically be refreshed.
+
+👨‍🏫 For a complete introductory example, see our [basic usage notebook](/examples/basic_usage.ipynb).
+It shows you how to store all types of items: python lists and dictionaries, `numpy` arrays, `scikit-learn` fitted models, `matplotlib`, `altair`, and `plotly` figures, etc.
+The resulting `skore` report has been exported to [this HTML file](https://sylvaincom.github.io/files/probabl/skore/basic_usage.html).
+
+## 🔨 Contributing
+
+Thank you for your interest!
+See [CONTRIBUTING.md](/CONTRIBUTING.md).
```

Comment:
Please put hard links when referencing the repository (`https://github.com/probabl-ai/skore/blob/main/CONTRIBUTING.md`), otherwise the link will be broken on PyPI.